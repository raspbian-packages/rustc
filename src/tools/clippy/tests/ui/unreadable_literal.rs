#![feature(plugin)]
#![plugin(clippy)]
#[warn(unreadable_literal)]
#[allow(unused_variables)]
fn main() {
    let good = (0b1011_i64, 0o1_234_u32, 0x1_234_567, 1_2345_6789, 1234_f32, 1_234.12_f32, 1_234.123_f32, 1.123_4_f32);
    let bad = (0b10110_i64, 0x12345678901_usize, 12345_f32, 1.23456_f32);
}
