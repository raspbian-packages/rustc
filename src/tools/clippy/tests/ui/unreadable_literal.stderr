error: long literal lacking separators
 --> $DIR/unreadable_literal.rs:7:16
  |
7 |     let bad = (0b10110_i64, 0x12345678901_usize, 12345_f32, 1.23456_f32);
  |                ^^^^^^^^^^^
  |
  = note: `-D unreadable-literal` implied by `-D warnings`
  = help: consider: 0b1_0110_i64

error: long literal lacking separators
 --> $DIR/unreadable_literal.rs:7:29
  |
7 |     let bad = (0b10110_i64, 0x12345678901_usize, 12345_f32, 1.23456_f32);
  |                             ^^^^^^^^^^^^^^^^^^^
  |
  = help: consider: 0x123_4567_8901_usize

error: long literal lacking separators
 --> $DIR/unreadable_literal.rs:7:50
  |
7 |     let bad = (0b10110_i64, 0x12345678901_usize, 12345_f32, 1.23456_f32);
  |                                                  ^^^^^^^^^
  |
  = help: consider: 12_345_f32

error: long literal lacking separators
 --> $DIR/unreadable_literal.rs:7:61
  |
7 |     let bad = (0b10110_i64, 0x12345678901_usize, 12345_f32, 1.23456_f32);
  |                                                             ^^^^^^^^^^^
  |
  = help: consider: 1.234_56_f32

error: aborting due to 4 previous errors

